R/generate_template.R

```diff
@@ -85,7 +90,7 @@
 #' create_new_application(name = 'mytestapp', location = tempdir())
 #'
```

Comment:
add an example with the right sidebar set to a valid custom icon.

Reply:
need to add a blank line before it.
<img width="657" alt="Screen Shot 2019-11-18 at 1 19 50 PM" src="https://user-images.githubusercontent.com/9085025/69095006-53c93c00-0a06-11ea-96f6-37f68612a00b.png">